src/main/java/edu/unc/lib/boxc/migration/cdm/services/CdmIndexService.java

```diff
@@ -295,12 +313,19 @@ private void indexObject(Connection conn, List<String> exportFieldValues)
      * @param force
      * @throws IOException
      */
-    public void createDatabase(boolean force) throws IOException {
+    public void createDatabase(boolean force, CdmIndexOptions options) throws IOException {
         ensureDatabaseState(force);
+        List<String> exportFields;
+
+        if (options.getCsvFile() != null) {
```

Comment:
I think it's okay to have the same behavior for CSV or CDM. CSV fields should be loaded with `loadFieldsFromProject` in either case, and I think `exportFields.addAll(MIGRATION_FIELDS);` is fine in both cases too.